packages/card/src/FzCard.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <section :class="[sectionStaticClass, backgroundColor]">
-    <header :class="[headerContainerComputedClass, borderColor]" @click="toggleOpen">
+    <header
+      :class="[headerContainerComputedClass, borderColor]"
+      @click="toggleOpen"
+      v-if="!props.hideHeader"
```

Comment:
Una cosa del genere potrebbe funzionare?

```suggestion
      v-if="(props.title || slots.header || slots['header-content'])"
```